services/web/server/src/simcore_service_webserver/computation_comp_tasks_listening_task.py

```diff
@@ -73,6 +75,15 @@ async def _update_project_outputs(
     )
 
     await projects_api.notify_project_node_update(app, project, node_uuid)
+    # get depending node and notify for these ones as well
+    depending_node_uuids = await project_get_depending_nodes(project, node_uuid)
```

Comment:
so now the notification is not only to the updated node but alls all dependent ones

what is the failing policy on notifications? Should we ignore failures if something went wrong? I guess it is not critical and the notification is periodic

The problem with creating this hooks is that at this side there is no control of what the callback is going to do.

Example, the callback could wait forever ... blocking this call -> should have a timeout the notifications?